plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -581,4 +582,17 @@ public DeltaLakeConfig setMetadataParallelism(int metadataParallelism)
         this.metadataParallelism = metadataParallelism;
         return this;
     }
+
+    public boolean isVacuumLoggingEnabled()
+    {
+        return vacuumLoggingEnabled;
+    }
+
+    @Config("delta.vacuum.logging.enabled")
```

Comment:
Why do we need a specific configuration for this ?

I've landed on https://github.com/delta-io/delta/commit/3ac120fe9f2d23e6debfccbfa1fbfb19d3d8ad5a 
Maybe add this commit to description of the PR for reference.

BTW Trino does support concurrent writes on all the file systems. (except HDFS)

Reply:
@ebyhr : Pleae comment. I think the reasoning is that in Spark this is configurable and disabled by default.

Reply:
I suggested adding this property for compatibility with Spark and also a kill-switch. 

The current implementation still has possibility of conflicts. For instance, the logging may fail if there is a concurrent operation because we don't have a retry logic in vacuum logic.